ModularTegustation/tegu_items/lc13/refinery/crates.dm

```diff
@@ -0,0 +1,70 @@
+/obj/structure/lootcrate
+	name = "Crate"
+	desc = "A crate recieved from a company"
+	icon = 'ModularTegustation/Teguicons/refiner.dmi'
+	icon_state = "crate_lcb"
+	anchored = FALSE
+	density = TRUE
+	resistance_flags = INDESTRUCTIBLE
+	var/lootlist =	list(
+		/obj/item/toy/plush/beeplushie, //higher values are more common
+		/obj/item/toy/plush/blank,
+		/obj/item/toy/plush/yisang,
+		/obj/item/toy/plush/faust,
+		/obj/item/toy/plush/don,
+		/obj/item/toy/plush/ryoshu,
+		/obj/item/toy/plush/meursault,
+		/obj/item/toy/plush/honglu,
+		/obj/item/toy/plush/heathcliff,
+		/obj/item/toy/plush/ishmael,
+		/obj/item/toy/plush/rodion,
+		/obj/item/toy/plush/sinclair,
+		/obj/item/toy/plush/outis,
+		/obj/item/toy/plush/gregor,
+		/obj/item/toy/plush/dante,
+		/obj/item/toy/plush/yuri)
+
+/obj/structure/lootcrate/attackby(obj/item/I, mob/living/user, params)
+	..()
+	if(I.tool_behaviour == TOOL_CROWBAR)
+		var/loot = pick(lootlist)
+		to_chat(user, "<span class='notice'>You open the crate!</span>")
+		new loot(get_turf(src))
+		qdel(src)
+
+/obj/structure/lootcrate/lingus
```

Comment:
It's best to keep type path names formal and understandable.
There is already enough issues with finding all-around-helper's records paper when its path is named "reddit".